enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -68,6 +80,12 @@ def add_arguments(self, parser):
             type=int,
         )
 
+    def _get_enterprise_customer_user_ids(self):
```

Comment:
This seems to be a static method.

Reply:
It would be better to make it static or go with a more descriptive approach as was being used previously.